docs/apache-airflow/core-concepts/tasks.rst

```diff
@@ -76,7 +76,6 @@ The possible states for a Task Instance are:
 * ``queued``: The task has been assigned to an Executor and is awaiting a worker
 * ``running``: The task is running on a worker (or on a local/synchronous executor)
 * ``success``: The task finished running without errors
-* ``shutdown``: The task was externally requested to shut down when it was running
```

Comment:
Should a note be added below this list to mention there used to be a shutdown state but not anymore?

Reply:
I felt like it's not useful information for the typical audience of the docs because the docs are meant for specific Airflow version and if someone has older version they should look at relevant, older docs. Probably should show in release notes. But I can modify it if you're sure.

Reply:
We discussed it in https://github.com/apache/airflow/pull/33746 and it turned out that SHUTDOWN not only has been deeply buried and nearly forgotten but also it has never been working for things like API. So I think I would prefer to pretend it never existed :P